src/lib.rs

```diff
@@ -359,6 +359,16 @@ const MAX_UNDECRYPTABLE_PACKETS: usize = 10;
 
 const RESERVED_VERSION_MASK: u32 = 0xfafafafa;
 
+// The default size of the receiver connection flow control window.
+const DEFAULT_CONNECTION_WINDOW: u64 = 48 * 1024;
```

Comment:
Should the default really be based on `set_initial_max_data()`?

Reply:
See my comment on DEFAULT_STREAM_WINDOW above.